content/en/dashboards/widgets/wildcard.md

```diff
@@ -41,6 +42,22 @@ However, if none of the Datadog widgets meets your visualization needs, a Wildca
 1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
 1. Click **Save**.
 
+#### Formulas Scalar vs. Formulas Timeseries
+
+In Datadog dashboards, scalar and timeseries visualizations serve different purposes and are used to represent data in distinct ways.
+
+**Timeseries**
+: This visualization is designed to display how your data changes over time.
+   - **Use-cases**: It's ideal for monitoring metrics that fluctuate, such as CPU usage, memory consumption, or request rates. It helps identify trends, patterns, and anomalies over a specified time range. 
+   - **Appearance**: These visualizations are typically line or area charts. They show datapoints plotted against time (usually on the x-axis), to observe changes and trends.
```

Comment:
I'm not sure we need to keep this section (since we are trying to focus convey that the request type is something that is independent of the visualization type - timeseries queries also can be used in the Heatmap widget), but if we do, DD timeseries are drawn in bar mode too (likely more frequently than areas).